.prettierignore

```diff
@@ -70,3 +70,24 @@ packages/framework/data-object-base/es5
 
 # Generated by policy-check
 packages/runtime/test-runtime-utils/src/assertionShortCodesMap.ts
+
+# Ignore all HTML files in docs since they're go templates
+docs/**/*.html
+
+# Ignore docs markdown files since those are formatted using markdownlint
+docs/**/*.md
+
+# github config yaml files are prone to breaking when reformatted
+.github/**/*.yml
+
+# Autogenerated file
+.github/fabricbot.json
+
+# These are actually templates, not pure YAML files
+server/**/templates/*.yaml
+
+# These files are auto-generated according to the comments in the files
+**/charts/**/Chart.yaml
+
+# Kubernetes
+**/kubernetes/**
```

Comment:
Any particular reason for this one? I've dealt with those files (/server/routerlicious/kubernetes) and I think the yaml and markdown ones wouldn't mind auto-formatting (templates excluded, but that's handled above already).

Reply:
I'll double check this.